README.md

```diff
@@ -2,19 +2,28 @@
 
 ⚠️ **Warning: this project is still very alpha**
 
-This project is:
+React component variations are:
```

Comment:
Technically the variations are just part of the descriptor object that variation providers return.